build/configs/artik053/tools/openocd/artik053_romfs.cfg

```diff
@@ -0,0 +1,255 @@
+source [find artik053-usb-ftdi.cfg]
```

Comment:
Do we need separate CFG file ? This approach may demand the possibility of having new *.cfg file for individual partitions. Kindly re-think about it.

Reply:
Yeah. but this commit if for ROMFS. We can think about it.

Reply:
This is not related to ROMFS. We discuss later.